src/components/button/button.tsx

```diff
@@ -75,6 +74,14 @@ export class Button
   /**  When `true`, interaction is prevented and the component is displayed with lower opacity. */
   @Prop({ reflect: true }) disabled = false;
 
+  /**
+   * The ID of the form owner to associate this component with.
```

Comment:
Maybe "The ID of the form owner to associate with the component"? IIRC ending a sentence with "with" is frowned upon by  the dictionary dwellers. Applies to the other components as well.

Reply:
Great suggestion with!